api/v6/products.thrift

```diff
@@ -40,6 +41,7 @@ struct Product {
   6: bool              accessible,        // Indicates whether the current user can access this product.
   7: bool              administrating     // Indicates that the current user can administrate the product.
   8: shared.DBStatus   databaseStatus     // Indicates the database backend status.
+  9: i64               runLimit           // Number of allowed runs for this product.
```

Comment:
Most of the other structs have `,` after the last element so adding a new member does not make the previous line modified.